provider/pkg/gen/schema.go

```diff
@@ -68,6 +68,10 @@ func PulumiSchema(swagger map[string]interface{}) pschema.PackageSpec {
 					Description: "If present, the default namespace to use. This flag is ignored for cluster-scoped resources.\n\nA namespace can be specified in multiple places, and the precedence is as follows:\n1. `.metadata.namespace` set on the resource.\n2. This `namespace` parameter.\n3. `namespace` set for the active context in the kubeconfig.",
 					TypeSpec:    pschema.TypeSpec{Type: "string"},
 				},
+				"deleteUnreachable": {
+					Description: "If present and set to true, the provider will delete resources on an unreachable kubernetes cluster",
```

Comment:
I'd suggest rewording this slightly since we aren't actually deleting the resources on the cluster.
```suggestion
					Description: "If present and set to true, the provider will delete resources associated with an unreachable kubernetes cluster from Pulumi state",
```

Reply:
```suggestion
					Description: "If present and set to true, the provider will delete resources on an unreachable Kubernetes cluster",
```